Corrected source.extension.vsixmanifest

mcquiggd opened the pull request:
Added support for Visual Studio 2019 versions, and Enterprise edition

mcquiggd committed:
Corrected source.extension.vsixmanifest

Added support for Visual Studio 2019 versions, and Enterprise edition

aberus commented on StackOverflowQuickLaunch/source.extension.vsixmanifest:
```diff
@@ -11,11 +11,9 @@
-    <InstallationTarget Version="14.0" Id="Microsoft.VisualStudio.Community" />
-    <InstallationTarget Version="15.0" Id="Microsoft.VisualStudio.Community" />
-    <InstallationTarget Version="16.0" Id="Microsoft.VisualStudio.Community" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[12.0, 17.0]"/>
```
This extensions has support from VS 2012 (version 11.0) and version 17.0 should be omitted, because we don't the know if this extension will work in a future major version of VS. 

```suggestion
    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[12.0, 17.0)" />
    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[11.0, 17.0)" />
    <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[11.0, 17.0)" />
```

mcquiggd replied:
Actually, the range values are "up to but not including 17.0". I have tested it using 16.0, and it will not work with updated copies of 2019. I have added another commit to standardise the layout, and include 11.0 - 17.0 (VS 2012 -2019)

https://devblogs.microsoft.com/visualstudio/how-to-upgrade-extensions-to-support-visual-studio-2019/

mcquiggd committed:
Altered support Visual Studio version range to include 11

The version range is up to but not including 17.0, according to Mads Kristensen:

https://devblogs.microsoft.com/visualstudio/how-to-upgrade-extensions-to-support-visual-studio-2019/

aberus committed:
Fixed spacing
